src/test/play-26/uk/gov/hmrc/hmrcfrontend/views/helpers/hmrcReportTechnicalIssueHelperPlay26Spec.scala

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.views.helpers
+
+import org.jsoup.Jsoup
+import org.scalatest.{Matchers, WordSpecLike}
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.Application
+import play.api.i18n.Lang
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import uk.gov.hmrc.hmrcfrontend.MessagesSupport
+import uk.gov.hmrc.hmrcfrontend.views.html.helpers._
+
+class hmrcReportTechnicalIssueHelperPlay26Spec
+    extends WordSpecLike
+    with Matchers
+    with MessagesSupport
+    with GuiceOneAppPerSuite {
+
+  override def fakeApplication(): Application =
```

Comment:
I think we can remove this now because we are setting it up manually in each test.

Reply:
Removed.  Thanks.